### DISCUSSION TITLE
Web view fails with ERR_TOO_MANY_REDIRECTS

### DISCUSSION THREAD

**Discussion by kasalalpay** [GitbookIO/community#461] [0. General bugs and questions]
ERR_TOO_MANY_REDIRECTS

![image](https://github.com/GitbookIO/community/assets/153176690/073ba5ef-94ab-4832-9c54-6f48832a8253)

At https://superfan.gitbook.io/blog/

I tried a force refresh in chrome, incognito in chrome, and tried edge. I deleted cookies. I also waited about 30 min to let changes propagate. Still, I cannot get to my published test collection. I plan to use Gitbook like something of a blog for project updates.

When I go to the base url, it redirects to https://app.gitbook.com/o/Uye4zMX9F6MwDZO7WqLV/s/QqZmmIWuy8hawHA9yuaT/ , shows a preloader for a very long time, then asks me to login to gitbook again. I never make it to the public web view.

Any ideas? TIA.

**Comment by aikoii** [comment]
Hey @kasalalpay I've checked https://superfan.gitbook.io/blog/ and it's working fine now. Have you managed to resolve the issue on your own? By the way, when I tried visiting https://app.gitbook.com/o/Uye4zMX9F6MwDZO7WqLV/s/QqZmmIWuy8hawHA9yuaT, I noticed that the space was deleted

**Reply by kasalalpay** [reply]
Hi, thanks for the response. I basically started over, turning off the 'share collection to web' on everything didn't help fyi, I deleted everything and started over. I'm not sure if that's useful info for you or not, maybe I should have left it as is to demonstrate the problem. I'd say this is resolved bc I started over, but I did not find the reason for the initial problems. Thanks again.

**Reply by aikoii** [reply]
Thanks for the feedback! It can be tricky now to pinpoint what happened, but it's good to know everything is working 🙂.
